vm/src/obj/objcomplex.rs

```diff
@@ -272,12 +266,12 @@ impl PyComplex {
             }
         };
 
-        let imag = match imag {
-            OptionalArg::Missing => Complex64::new(0.0, 0.0),
-            OptionalArg::Present(obj) => {
+        let imag = match args.imag {
+            None => Complex64::new(0.0, 0.0),
+            Some(obj) => {
                 if let Some(c) = try_complex(&obj, vm)? {
                     c
```

Comment:
You can say `&vm.ctx.types.str_type` instead of `&PyString::class()`

Reply:
Fixed it!